ruff.toml

```diff
@@ -3,4 +3,4 @@ line-length = 120
 target-version = "py310"
 
 [lint]
-ignore = ["E741"]
+select = ["E4", "E7", "E9", "F", "W", "I", "ASYNC"]
```

Comment:
Default ruleset is
select = ["E4", "E7", "E9", "F"]
So this add warning (W), isort (I), and ASYNC rules